Orbitersdk/samples/ProjectApollo/src_sys/toggleswitch.cpp

```diff
@@ -487,7 +487,7 @@ void TwoPositionSwitch::SaveState(FILEHANDLE scn)
 {
 	char buffer[1000];
 
-	sprintf(buffer, "%i %u", state, GetFlags()); 
+	sprintf(buffer, "%i %i %u", state, Failed, GetFlags()); 		//Saves Failed attribute in second field
```

Comment:
You can't change the position of already existing fields. That will break all existing saves.
If you need to introduce a new field, append it instead of inserting. Ideally this would be part of the flags instead of its own thing though.

Reply:
I tried adding "fs.Failed = Failed ? 1 : 0;" to GetFlags() and also added it to ToggleSwitchFlags struct, but it only prints 2 fields, what I assume are state and held.